### PR TITLE
Develop/bff support

### PR DESCRIPTION
**Pull request by Patterbear**
- Program is now working platform-independently with .doc and .xls files.

- It works by wiping macro streams and replacing them with empty bytes.

- Unfortunately, the 'ppt' format is proving to be hard to crack, as it stores its macros in a well-embedded way within the 'PowerPoint Document' branch.

- I will open a new issue for this and open it up for help.

### CODE REVIEW

File: ms_office_macro_bleach/bleach.py [hunk 1]
```diff
@@ -60,10 +73,53 @@ def unzip_file(file):
...
+            ole.write_stream(macro_stream, bytes(bytearray(macro_stream_size)))
+        ole.close()
+
+        if len(macro_streams) > 0:
+            macros_found = True
+
+    if file_type == "ppt":
+        streams = OleFileIO(file).listdir(streams=True)
+        print(streams)
+        print("GET THIS WORKING")
```
Comment (rothoma2) [inline ms_office_macro_bleach/bleach.py:112]: Maybe change this for a print like "unsupported" or similar.

File: ms_office_macro_bleach/bleach.py [hunk 2]
```diff
@@ -60,10 +73,53 @@ def unzip_file(file):
 
 
 def remove_macros(file, notify):
+    file_type = file.split(".")[-1].lower()
+
```
Comment (rothoma2) [inline ms_office_macro_bleach/bleach.py:77]: I really like this coding style, is very clean and readable :)

**Comment by rothoma2** [comment]
I didnt notice changes to the tests. Do u already have tests for the new formats or plan to add them later?

**Comment by Patterbear** [comment]
> I didnt notice changes to the tests. Do u already have tests for the new formats or plan to add them later?

I was going to do a new test branch for them

File: ms_office_macro_bleach/bleach.py (hunk 2 above)
Comment (Patterbear) [reply ms_office_macro_bleach/bleach.py:77]: Thank you :)

I used to cram everything on one line without considering readability, however with clean code principles I'm making the code more readable.

**Comment by rothoma2** [comment]
> > I didnt notice changes to the tests. Do u already have tests for the new formats or plan to add them later?
> 
> I was going to do a new test branch for them

Sure we can add the tests later.

**Review by rothoma2 — APPROVED** [review]

**Pull request merged**